de.fu_berlin.inf.dpp.intellij/META-INF/plugin.xml

```diff
@@ -35,16 +35,17 @@
 
     <actions>
         <!-- Add your actions here -->
-        <group id="saros.MainMenu" text="Saros" description="Saros menu">
+        <group id="saros.MainMenu" text="S_aros" description="Saros menu">
             <add-to-group group-id="MainMenu" relative-to-action="HelpMenu" anchor="before"/>
             <action id="saros.about"
                     class="de.fu_berlin.inf.dpp.intellij.ui.menu.AboutSarosHandler"
-                    text="About Saros"/>
+                    text="_About Saros"/>
         </group>
 
         <group keep-content="true" compact="false" popup="true"
                id="saros.ShareWith"
-               icon="/icons/famfamfam/session_tsk.png" text="Share With..."
+               icon="/icons/famfamfam/session_tsk.png" text="_Share With..."
+               description="Share this modules through Saros"
```

Comment:
Grammar: Either "theses modules" or "this module"
Wording: "with Saros"?

Reply:
Fixes, changed to "this module".
I don't like the wording "with Saros" as I always here it in the context of "Share the module with user XYZ", which would suggest to me that we are trying to share the module with a user named "Saros".

How about "Share this module using Saros"?

Reply:
If I'm not mistaken, that's the difference between *ablative of manner* and *ablative of accompaniment* (language nerds, [this way][1]).

Your suggestion "using Saros" is fine.

[1]: https://latin.stackexchange.com/questions/608/fighting-with-someone-and-the-different-uses-of-with-in-latin